test/Elastic.Apm.Tests/SerializationTests.cs

```diff
@@ -129,6 +129,26 @@ public void DbStatementLengthTest()
 			Assert.Equal(str, deserializedDb.Statement);
 		}
 
+		/// <summary>
+		///	Creates a service instance with a name that is longer than <see cref="Consts.PropertyMaxLength"/>.
+		/// Makes sure the name is truncated.
+		/// </summary>
+		[Fact]
+		public void ServiceVersionLengthTest()
```

Comment:
Shouldn't this be `ServiceNameLengthTest` instead of `ServiceVersionLengthTest`?

Reply:
Hope it's ok that I renamed it 👼  

That way I can merge it now - before we release.